src/Raven.Client/Util/JavascriptConversionExtensions.cs

```diff
@@ -1008,6 +1008,129 @@ public override void ConvertToJavascript(JavascriptConversionContext context)
+                using (writer.Operation(methodCallExpression))
+                {
+                    var arg = methodCallExpression.Arguments[0];
+
+                    if (arg is UnaryExpression { Operand: LambdaExpression { Body: BinaryExpression { Left: MethodCallExpression mce } } })
+                    {
+                        var objReplace = (MemberExpression)mce.Object;
+                        var replaceExprassion = Expression.Parameter(objReplace.Expression.Type, _alias);
+                        var objReplace1 = objReplace.Update(replaceExprassion);
+                        var curmce = mce.Update(objReplace1, mce.Arguments);
+                        writer.Write("includes.document(");
+                        context.Visitor.Visit(curmce);
+                        writer.Write(")");
+                        return;
+                    }
+
+                    if (arg is UnaryExpression { Operand: LambdaExpression arg2 })
+                    {
+                        var targetExpression = (MemberExpression)arg2.Body;
+                        var replaceExprassion = Expression.Parameter(targetExpression.Expression.Type, _alias);
+                        targetExpression = targetExpression.Update(replaceExprassion);
+
+                        writer.Write("includes.document(");
+                        context.Visitor.Visit(targetExpression);
+                        writer.Write(")");
+                        return;
+                    }
+                    throw new NotSupportedException("You can't use the include that way try: let _ = RavenQuery.Include<T>(Expression<Func<T, string>> path)");
+                }
+            }
+        }
+
+        public class IncludeCountersAndTimeSeriesSupport : JavascriptConversionExtension
+        {
+            public static readonly IncludeCountersAndTimeSeriesSupport Instance = new IncludeCountersAndTimeSeriesSupport();
+
+            public override void ConvertToJavascript(JavascriptConversionContext context)
+            {
+                var methodCallExpression = context.Node as MethodCallExpression;
+
+                if (methodCallExpression == null)
+                    return;
+                if (methodCallExpression.Method.DeclaringType != typeof(RavenQuery))
+                    return;
+                
+                var methodName = methodCallExpression.Method.Name;
+                var hasCounters = methodName == nameof(RavenQuery.IncludeCounter) ||
+                                 methodName == nameof(RavenQuery.IncludeCounters) ||
+                                 methodName == nameof(RavenQuery.IncludeAllCounters);
+
+                var hasTimeSeries = methodName == nameof(RavenQuery.IncludeTimeSeries);
+
+                    if (hasCounters == false && hasTimeSeries == false)
+                    return;
+
+                var expectedArgsCount = methodName == nameof(RavenQuery.IncludeAllCounters) ? 1 : 2;
```

Comment:
make a variable from `methodName == nameof(RavenQuery.IncludeAllCounters)` (like `hasCounters`) and use it here and in `hasCounters`

Reply:
done